business-registry-dashboard/app/pages/[encodedOrgId]/affiliationInvitation/acceptToken/[token].vue

```diff
@@ -0,0 +1,86 @@
+<script setup lang="ts">
+import { useRoute } from 'vue-router'
+
+const { t } = useI18n()
+const accountStore = useConnectAccountStore()
+const affStore = useAffiliationsStore()
+const brdModal = useBrdModals()
+const route = useRoute()
+const toast = useToast()
+
+// Token parsing
+const parseToken = (encodedToken: string): AffiliationToken => {
+  try {
+    const tokenObject = encodedToken.split('.')[0]
+    const decoded = atob(tokenObject as string)
+    return JSON.parse(decoded)
+  } catch (error) {
+    console.error('Failed to parse token:', error)
+    throw new Error('Invalid token format')
+  }
+}
+
+definePageMeta({
+  checkMagicLink: true,
+  order: 0
+})
+
+onMounted(async () => {
+  try {
+    const token = parseToken(route.params.token as string)
+    const orgId = token.fromOrgId
+    if (orgId === accountStore.currentAccount.id) { // Check if the account is the same as the one that initiated the request
+      // Parse the URL and try to add the affiliation
+      parseUrlAndAddAffiliation(token, route.params.token as string)
+      // Load affiliations to update the table
+      await affStore.loadAffiliations()
+    } else {
+      toast.add({ title: 'Please login with the account that initiated the request.' })
```

Comment:
Try not to break the i18n, it's hard to unwind if it starts. Or put in a bog TODO: to fix it.
We are working with OCM, etc. for them to edit MD and language files ... so we sweat the details a little less as they can edit and preview in the temp URLs

Reply:
Yep, makes a lot of sense. Done, this has been removed altogether. Will make sure to keep things clean moving forward.